### PR TITLE
bugfx/codeexample > cpp added; else if added for html bug; color update

### PR DESCRIPTION
**Pull request by jeesunikim**
[Confirm <CodeExample/> coloring for scss, tx, toml](https://app.asana.com/0/1119309091112078/1168246626754913)
[<CodeExample/> bug with html code snippet](https://app.asana.com/0/1119309091112078/1168028367950531)

`cpp` is the right term instead of `c++`

### CODE REVIEW

File: content/docs/glossary/versioning.mdx [hunk 1]
```diff
@@ -26,21 +31,29 @@ A node considers a step invalid either because they do not understand it or some
...
 The reason for this is that nodes must be able to replay transactions from history (down to version 'n'), yet there might be some issue/vulnerability that we don't want to be exploitable for new transactions.
 
 ### Ledger object versioning
+
 Data structures that are likely to evolve over time contain the following extension point:
-```C++
+
+<CodeExample>
+
+```cpp
```
Comment (jeesunikim) [inline content/docs/glossary/versioning.mdx]: @rice2000, Prism recommends `cpp` for `C++`. See under "Supported languages" on https://prismjs.com/

File: src/helpers/documentation.js [hunk 2]
```diff
@@ -175,7 +175,8 @@ export const buildDocsContents = (data, rootDir) => {
     const firstTopic = topic.nodes[0];
     const relPath = buildRelPath(topicPath, rootDir);
     const topicId = firstTopic.id;
-    const topicOrder = firstTopic.fields.metadata.data.order;
+    const topicOrder =
+      firstTopic.fields && firstTopic.fields.metadata.data.order;
```
Comment (vcarl) [inline src/helpers/documentation.js]: Could also do `firstTopic.fields?.metadata.data.order` 👀

Comment (jeesunikim) [reply src/helpers/documentation.js]: 🔥

Comment (jeesunikim) [reply src/helpers/documentation.js]: Imma go `findAll` replace `&&`

Comment (jeesunikim) [reply src/helpers/documentation.js]: Updated! @vcarl

**Pull request merged**